railties/test/application/configuration_test.rb

```diff
@@ -1613,6 +1613,24 @@ class Post < ActiveRecord::Base
       assert ActiveRecord::ConnectionAdapters::SQLite3Adapter.represent_boolean_as_integer
     end
 
+    test "represent_boolean_as_integer should be able to set via config.active_record.sqlite3_represent_boolean_as_integer" do
+      remove_from_config '.*config\.load_defaults.*\n'
+
+      app_file "config/initializers/new_framework_defaults_5_2.rb", <<-RUBY
+        Rails.application.config.active_record.sqlite3_represent_boolean_as_integer = true
+      RUBY
+
+      app_file "app/models/post.rb", <<-RUBY
+        class Post < ActiveRecord::Base
+        end
+      RUBY
+
+      app "development"
+      Post.object_id # force lazy load hooks to run
```

Comment:
Maybe we should just include a `lazy_load { Post }` method everywhere in our test suite that basically no ops?

```ruby
def lazy_load
  yield # Tasty clarifying sugar, homie! We only need to reference a constant to load it.
end
```